packages/cashier/src/Components/Form/payment-agent-deposit.jsx

```diff
@@ -20,7 +20,6 @@ const PaymentAgentDeposit = ({ onChangePaymentMethod, payment_agent_list, select
             <Text as='p' size='xs' weight='bold' color='prominent' className='payment-agent-list__list-header'>
                 <Localize i18n_default_text='Payment agents' />
             </Text>
```

Comment:
```suggestion
      <div className='payment-agent-list__list-header'>
            <Text as='p' size='xs' weight='bold' color='prominent' className='payment-agent-list__list-header-text'>
                <Localize i18n_default_text='Payment agents' />
            </Text>
            <div className='payment-agent-list__list-header-line' />
      </div>
```
Would this be a more cleaner approach than using `::after` with negative margins here.

Reply:
I use `div` instead of `::after`, remove unneeded styles and css classes.